### ISSUE TITLE
Multiple tests run fails with Akka.TestKit 1.4.8 (worked fine with 1.4.7)

### ISSUE THREAD

**Issue by snekbaev** [akkadotnet/akka.net#4496]
Hi,

Akka: 1.4.8, NUnit3 package, Windows 10 Pro, .NET Framework 4.7.2, using Resharper to run tests.

Just upgraded from 1.4.7 to 1.4.8 and noticed that tests are failing with:
```
System.NullReferenceException : Object reference not set to an instance of an object.
TearDown : System.NullReferenceException : Object reference not set to an instance of an object.
   at Akka.TestKit.TestKitBase.<>c.<InitializeTest>b__10_1(Config c)
   at Akka.Util.Option`1.Select[TNew](Func`2 selector)
   at Akka.TestKit.TestKitBase.InitializeTest(ActorSystem system, ActorSystemSetup config, String actorSystemName, String testActorName)
   at Akka.TestKit.NUnit3.TestKit.InitializeActorSystemOnSetUp()
--TearDown
   at Akka.TestKit.TestKitBase.Dilated(TimeSpan duration)
   at Akka.TestKit.TestKitBase.Shutdown(ActorSystem system, Nullable`1 duration, Boolean verifySystemShutdown)
   at Akka.TestKit.TestKitBase.Shutdown(Nullable`1 duration, Boolean verifySystemShutdown)
   at Akka.TestKit.NUnit3.TestKit.AfterAll()
```
tests pass if I run one at a time, but run fails with several tests selected within such run.

If Akka.TestKit is rolled back to 1.4.7 then everything's back to normal.

Thank you!

**Comment by Aaronontheweb** [comment]
This is probably related to the changes I made to the TestKit to introduce `ActorSystemSetup` overloads to the constructors.

**Comment by Aaronontheweb** [comment]
found a reproduction in one of our own libraries here https://github.com/akkadotnet/Akka.DI.AutoFac/pull/68

**Comment by alexvaut** [comment]
Hi, do you think there is a workaround until this is fixed ? Thanks.

**Comment by stefanzueger** [comment]
Hi @Aaronontheweb, do you know if there is a workaround for the problem until there is a fix?

**Comment by Aaronontheweb** [comment]
@stefanzueger I'll assign this to @Arkatufus for the current sprint in progress.

**Comment by Arkatufus** [comment]
Should be fixed with #4603
